radboudumc-theme/sass/_typography.scss

```diff
@@ -74,29 +78,44 @@ div.page-content > p {
   div.container > ul,
   div.page-content > p,
   div.page-content > ol,
-  div.page-content > ul {
+  div.page-content > ul,
+  div.news-item > p {
     padding-right: 5em;
   }
+  div.news-item img {
+    width: 700px;
+    max-width: unset;
+  }
 }
 @media (min-width: 992px) {
   div.container > p,
   div.container > ol,
   div.container > ul,
   div.page-content > p,
   div.page-content > ol,
-  div.page-content > ul {
+  div.page-content > ul,
+  div.news-item > p {
     padding-right: 10em;
   }
+  div.news-item img {
+    width: 900px;
+    max-width: unset;
+  }
 }
 @media (min-width: 1200px) {
   div.container > p,
   div.container > ol,
   div.container > ul,
   div.page-content > p,
   div.page-content > ol,
-  div.page-content > ul {
+  div.page-content > ul,
+  div.news-item > p {
     padding-right: 20em;
   }
+  div.news-item img {
+    width: 1000px;
```

Comment:
Not all images are large enough to be displayed at 1000px, probably many of them aren't. E.g: https://www.computationalpathologygroup.eu/highlights/defense-maschenka-balkenhol/

Can't we use max-width here so that it works for all sizes? So use max-width 1000 and 900 for these breakpoints? 

Btw, maybe I missed this during the last reviews, can the breakpoints use the bootstrap breakpoints instead of hardcoded pixel widths?